k8s/ceramic/manifests/bootstrap.yaml

```diff
@@ -16,15 +15,15 @@ spec:
           - "/usr/bin/runner"
           - "bootstrap"
         env:
+          - name: RUNNER_OTLP_ENDPOINT
+            value: 'http://otel:4317'
           - name: RUST_LOG
-            value: 'info,runner=debug'
+            value: 'info,runner=trace'
```

Comment:
Any reason why runner is at trace?

Reply:
Runner is the code we write so figured it was nice to have full verbosity there. Obviously its easy to change.